docs/reference/mapping/types/semantic-text.asciidoc

```diff
@@ -148,3 +182,5 @@ In bulk requests, all fields that are copied to a `semantic_text` field must hav
 
 * `semantic_text` fields are not currently supported as elements of <<nested,nested fields>>.
 * `semantic_text` fields can't be defined as <<multi-fields,multi-fields>> of another field, nor can they contain other fields as multi-fields.
+* When ingestion pipelines update an already existing `semantic_text` field value, inference won't be recalculated for the field automatically.
+The inference will only be recalculated when the document is reindexed.
```

Comment:
This isn't the problem. If an ingestion pipeline updates a `semantic_text` field directly, everything is fine. The issue is if an ingestion pipeline updates a field that is copied to a `semantic_text` field. In this case, the update is silently ignored and the indexed embedding becomes stale.

Reply:
Thanks for noticing this - I started updating the limitations but decided to include them in copy_to . Forgot to clean this up! Doing it now